airbyte-integrations/connectors/source-googleanalytics-singer/build.gradle

```diff
@@ -10,12 +10,6 @@ airbytePython {
 
 
 dependencies {
-    implementation files(project(':airbyte-integrations:bases:base-standard-source-test-file').airbyteDocker.outputs)
     implementation files(project(':airbyte-integrations:bases:base-singer').airbyteDocker.outputs)
```

Comment:
we don't need singer docker, right?

Reply:
ok, I think we can update this connector with the CDK package at least, so we will not use base docker images.